samples/web/content/apprtc/apprtc.py

```diff
@@ -366,12 +390,19 @@ def add_client_to_room(host, room_id, client_id, is_loopback):
       room.add_client(client_id, Client(is_initiator))
       if is_loopback:
         room.add_client(constants.LOOPBACK_CLIENT_ID, Client(False))
+      if allowed_clients is not None:
+        for allowed_client in allowed_clients:
+          room.add_allowed_client(allowed_client)
     else:
       is_initiator = False
       other_client = room.get_other_client(client_id)
       messages = other_client.messages
       room.add_client(client_id, Client(is_initiator))
       other_client.clear_messages()
+      # Check if the callee was the callee intended by the caller.
+      if not room.has_allowed_client(client_id):
+        error = constants.RESPONSE_INVALID_ROOM
```

Comment:
We should not allow to "accept" for a room not started with "call", which indicates a client side bug.